reviews/sandbox/2022/2022-Confidential-Containers-annual.md

```diff
@@ -0,0 +1,155 @@
+# Confidential Containers 2022 Annual Review
+
+## Background
+
+Confidential Containers is a CNCF Sandbox project that enables cloud native
+confidential computing by taking advantage of a variety of hardware platforms
+and technologies. The project was accepted as a CNCF Sandbox in March 2022.
+
+Confidential Containers builds on existing and emerging hardware security
+technologies such as Intel SGX, Intel TDX, AMD SEV and IBM Z Secure Execution,
+in combination with new software frameworks to help better secure user data in
+use for cloud native workloads.
+
+The main goal of the project is to standardize confidential computing at the
+container level and simplify its consumption in Kubernetes. This is in order to
```

Comment:
We use the term container or pod level interchangeably. Wondering if `pod level` is more accurate here ?

Reply:
pod level makes more sense, fixed as well.